judge/forms.py

```diff
@@ -69,13 +69,20 @@ def __init__(self, *args, **kwargs):
 
 class ProblemSubmitForm(ModelForm):
     source = CharField(max_length=65536, widget=AceWidget(theme='twilight', no_ace_media=True))
+    judge = ChoiceField(choices=(), widget=forms.HiddenInput(), required=False)
 
-    def __init__(self, *args, **kwargs):
+    def __init__(self, *args, judge_queryset=(), **kwargs):
```

Comment:
This is not actually a queryset? Name is pretty confusing...

Reply:
This has been renamed to `judge_choices`.